src/helpers/format/format.ts

```diff
@@ -442,6 +442,16 @@ export function numberToString(number: number, decimalSeparator: string): string
   return decimalDigits ? integerDigits + decimalSeparator + decimalDigits : integerDigits;
 }
 
+/**
+ * Check if the given date format represents a duration,
+ * such as "hh:mm", "hh:mm:ss", or "hhhh:mm:ss".
+ * It must include hours ("h") and minutes ("m"),
+ * and must not contain date parts ("d", "y") or time-of-day indicators like "a" (AM/PM).
+ */
+export function isDurationFormat(format: Format): boolean {
+  return format.includes("h") && format.includes("m") && !/[ady]/.test(format);
+}
+
```

Comment:
I think duration is only `.*hhhh.*`. See `formatJSDatePart`

`hh:mm:ss` is a time, and it doesn't make much sense to SUM them. `12:00:00` + `20:00:00` would give `08:00:00` which doesn't make that much sense. I guess you could argue that AVG make some sense in that case, but it's probably not worth a separate case.

With `hhhh`, we get `32:00:00` which make sense.